cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MessagingMetricsProcessorService.java

```diff
@@ -253,6 +253,8 @@ protected void run() {
           // 5-min initial delay and scheduled to run each min after that
           dataMigratorExecutor.scheduleAtFixedRate(dataMigrator, 5, 1, TimeUnit.MINUTES);
           LOG.info("Scheduled metrics migration thread for resolution INT_MAX, 3600, 60 tables");
+        } else {
+          LOG.info("All Metrics data have been migrated, not scheduling migration thread");
```

Comment:
I think this should be at most debug, if it should be logged at all. To the user it is only of interest if the migratuon was started, the first time after upgrade. Other than that, this message is only confusing